controllers/front/ProductController.php

```diff
@@ -1278,18 +1278,39 @@ public function getTemplateVarPage()
             $page['body_classes']['product-customizable'] = true;
         }
         $page['admin_notifications'] = array_merge($page['admin_notifications'], $this->adminNotifications);
+        $page['meta']['title'] = $this->getProductPageTitle();
```

Comment:
By comparing the previous behavior, didn't you expected to send `$page` as parameter?

Reply:
yes you're right, this function was made to work for the ajax case as well
I wanted to test if it worked without any argument but forgot to reuse the argument before commit, thank you

Reply:
fixed